src/Network/Network/help/New-AzExpressRouteGateway.md

```diff
@@ -67,6 +67,15 @@ ProvisioningState   : Succeeded
 The above will create a resource group, Virtual WAN, Virtual Network, Virtual Hub in West US in "testRG" resource group in Azure. 
 An ExpressRoute gateway will be created thereafter in the Virtual Hub with 2 scale units.
 
+### Example 2
+
+Creates a Scalable ExpressRoute Gateway. (autogenerated)
+
+<!-- Aladdin Generated Example -->
+```powershell
+New-AzExpressRouteGateway -MaxScaleUnits <UInt32> -MinScaleUnits 2 -Name 'testExpressRoutegw' -ResourceGroupName 'testRG' -Tag '/Sql' -VirtualHubName <String>
```

Comment:
The `Tag` parameter requires a hashtable for its value. The specified value is not a valid hashtable.